db/migrate/20150122124937_reslug_some_manual_sections.rb

```diff
@@ -0,0 +1,248 @@
+      original_document_id: emergency_publishing_id,
+      original_version_number: 2,
+      new_document_id: feedback_id,
+      new_version_number: 1,
+      new_slug: feedback_slug,
+    )
+
+    # Create a new document for the "Help with content quality" editions, and
+    # move them onto it
+    move_edition_to_document(
+      original_document_id: emergency_publishing_id,
+      original_version_number: 3,
+      new_document_id: content_quality_id,
+      new_version_number: 1,
+      new_slug: content_quality_slug,
+    )
+
+    move_edition_to_document(
+      original_document_id: emergency_publishing_id,
+      original_version_number: 4,
+      new_document_id: content_quality_id,
+      new_version_number: 2,
+      new_slug: content_quality_slug,
+    )
+
+    move_edition_to_document(
+      original_document_id: emergency_publishing_id,
+      original_version_number: 5,
+      new_document_id: content_quality_id,
+      new_version_number: 3,
+      new_slug: content_quality_slug,
+    )
+
+    move_edition_to_document(
+      original_document_id: emergency_publishing_id,
+      original_version_number: 6,
+      new_document_id: content_quality_id,
+      new_version_number: 4,
+      new_slug: content_quality_slug,
+    )
+
+    move_edition_to_document(
+      original_document_id: emergency_publishing_id,
+      original_version_number: 7,
+      new_document_id: content_quality_id,
+      new_version_number: 5,
+      new_slug: content_quality_slug,
+    )
+
+    # Move the "National emergency publishing guidelines" editions onto the
+    # national-emergency-publishing-guidelines document,
+    move_edition_to_document(
+      original_document_id: support_processes_id,
+      original_version_number: 2,
+      new_document_id: emergency_publishing_id,
+      new_version_number: 2,
+      new_slug: emergency_publishing_slug,
+    )
+
+    move_edition_to_document(
+      original_document_id: support_processes_id,
+      original_version_number: 3,
+      new_document_id: emergency_publishing_id,
+      new_version_number: 3,
+      new_slug: emergency_publishing_slug,
+    )
+
+    move_edition_to_document(
+      original_document_id: support_processes_id,
+      original_version_number: 4,
+      new_document_id: emergency_publishing_id,
+      new_version_number: 4,
+      new_slug: emergency_publishing_slug,
+    )
+
+    # Make a new edition of the manual and set the document_ids to be in the
+    # correct order and to not include the (now defunct)
+    # gov-uk-support-requests-processes-and-response-times edition
+    manual_record = ManualRecord.where(
+      manual_id: manual_id,
+    ).first
+
+    latest_manual_edition = manual_record.editions.last
+
+    current_document_order = latest_manual_edition.document_ids
+    new_document_order = current_document_order
+      .-([support_processes_id, emergency_publishing_id, feedback_id])
+      .+([emergency_publishing_id, content_quality_id, feedback_id])
```

Comment:
Any chance of switching to an explicit set order rather than diffing?  Makes it harder to keep track of what's happening, plus keeping track of the operator precedence.

For instance, if they do bind the way I think they do, the new IDs will be at the end of the list, is that what you're after?